app/javascript/src/components/Dashboard/Notes/List.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Typography, Dropdown, Button, Avatar } from "neetoui";
+
+import { calculateCreatedAgo, formatDateAndTime } from "./utils";
+
+const List = ({ notes }) => {
+  const { Menu, MenuItem } = Dropdown;
+  const listItems = ["Edit", "Delete"];
```

Comment:
- How does it make sense to name this `listItems`? Do you think it makes sense in the context of this component and what it's doing?
- Why is this not in a `constants.js` file?

Reply:
Fixed the issue.